integrations/pkg_config_pc_files.rst

```diff
@@ -47,16 +42,17 @@ Example of a ``.pc`` file with an absolute path:
     Libs: -L${libdir} -L${sharedlibdir} -lz
     Cflags: -I${includedir}
 
+To solve this problem there are different approaches that can be followed.
```

Comment:
Those are just alternative solutions, or do Conan prefer one over another? I would state that he preferred one is...

Reply:
Up to the user I would say. This describes just ways of intergating pc files with Conan